specs/src/specs/header-sync.md

```diff
@@ -0,0 +1,47 @@
+# P2P Header Sync
+
+## Abstract
+
+The P2P Header Sync is a p2pP exchange service for rollkit headers that implements the [go-header](https://github.com/celestiaorg/go-header) interface. The main components are listed in the table below.
+
+| Component  | Description                                                                                                                                                                |
+| ---------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
you don't need to match the width of the column, it will auto size, you just need 3 dashes

https://www.markdownguide.org/extended-syntax/#tables

```suggestion
| --- | --- |
```